### ISSUE TITLE
Track banned operators

### ISSUE THREAD

**Issue by redraskal** [redraskal/r6-dissect#85]

**Comment by CodingPenguin1** [comment]
@redraskal Is there a chance for this? I'm getting back into stats for my old collegiate team, and this would save a bunch of manual effort on their part

Down to help how I can, though I'm not familiar with Go, at least right now. Doesn't look too bad...

**Comment by redraskal** [comment]
> [@redraskal](https://github.com/redraskal) Is there a chance for this? I'm getting back into stats for my old collegiate team, and this would save a bunch of manual effort on their part
> 
> Down to help how I can, though I'm not familiar with Go, at least right now. Doesn't look too bad...

@CodingPenguin1 Yeah, I did some research this past hour and believe I found the data in the replay files.

Unfortunately, the operator IDs are different than the ones I currently use. I'm thinking about how to design a tool to pull these operator IDs from replay files.

I think the current IDs in r6-dissect are used explicitly for op icons in the game, while this ID might identify the operator.

**Comment by CodingPenguin1** [comment]
Thanks for taking the time to look into it! Do you have a write-up of the file structure as far as you've discovered so far?
